_toc.yml

```diff
@@ -0,0 +1,18 @@
+format: jb-book
+root: README
+parts:
+  - caption: Getting started
+    chapters:
+    - file: docs/get-started
+    - file: docs/how-to-contribute
+  - caption: AIOps projects
+    chapters:
+    - file: docs/aiops-projects
+    - file: docs/project-document-template
+    - file: docs/project-structure
+  - caption: Best Practices and Guidelines
+    chapters:
+    - file: docs/guidelines-for-reviewing-datascience-code.md
+    - file: docs/data-visualization-best-practices.md
```

Comment:
Also here updated it so that this does not have bold text on the jupyterbook sidebar